### ISSUE TITLE
Allow users to subscribe to inverse of any red flag list

### ISSUE THREAD

**Issue by fabacab** [fabacab/pat-okcupid#7]
This is a very important feature. If people create red flag question sets that, for instance, red flag feminists, then this feature would enable feminists to USE that list in order to red flag "people who are NOT feminists."

See also https://twitter.com/focalintent/status/331920686614450176

**Comment by fabacab** [comment]
We need to work out what a "positive highlight" would mean, as well. See #4 for related discussion.

**Comment by focalintent** [comment]
Yeah - I saw the FAADE data dump - ah, if only fetlife had a mechanism for allowing people to give more ordered/organized data about themselves that could be mined, like OKC's questions.  (Fetishes, perhaps?)

Also - I appreciated the cultural norms post when I first saw it posted (enjoyed sometimes feels like the wrong word for some of these conversations), thank you for writing that!

Also - back to this particular issue, I don't know how you prefer to do such things for your projects - there's no explicit issue for multiple lists of red flag questions (or for user edit-ability/sharing of such lists).  Is that assumed to be rolled up under here, or do you want separate issues for them?  (My process-brain insists on getting to type every now and again)
